corehq/ex-submodules/casexml/apps/phone/dbaccessors/sync_logs_by_user.py

```diff
@@ -11,3 +12,11 @@ def get_last_synclog_for_user(user_id):
         reduce=False,
         include_docs=True,
     ).one()
+
+
+def get_all_sync_logs_docs():
```

Comment:
Presumably this should only be used in tests. Might be worth adding an assert that `settings.UNIT_TESTING` is true.